### PR TITLE
Prevent spurious CI timeouts on closing the project in UI tests

### PR DESCRIPTION
**Pull request by PawelLipski**
For instance, see https://app.circleci.com/pipelines/github/VirtusLab/git-machete-intellij-plugin/3793/workflows/2b3434c2-9258-4305-9d60-b3bad59a342f/jobs/3979:

image

### CODE REVIEW

File: scripts/run-ui-tests [hunk 1]
```diff
@@ -5,14 +5,14 @@
...
 # or a field of `ext.intellijVersions` from build.gradle (e.g. `earliestSupportedMajor`).
-# If skipped, then `buildTarget` is assumed.
+# If skipped or empty, then `buildTarget` is assumed.
 
 # If <only-tests> is present and non-empty, only the UI test methods
 # whose names start with the given prefix will be executed (instead of all UI test methods).
 
 set -e -o pipefail -u
 
-intellij_version_key=${1-buildTarget}
```
Comment (PawelLipski) [inline scripts/run-ui-tests:15]: This was "assume `buildTarget` if $1 not defined"

File: scripts/run-ui-tests [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
+intellij_version_key=${1:-buildTarget}
```
Comment (PawelLipski) [inline scripts/run-ui-tests:15]: This is This was "assume `buildTarget` if $1 not defined *or defined&empty*", so now one can run `./scripts/run-ui-tests "" pull`

File: uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala [hunk 3]
```diff
@@ -60,7 +60,9 @@ class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH
   @After
   def afterEach(): Unit = {
     intelliJ.probe.awaitIdle()
-    intelliJ.probe.listOpenProjects.foreach(intelliJ.probe.closeProject)
+    // Note that we shouldn't wait for a response here (so we shouldn't call org.virtuslab.ideprobe.ProbeDriver#send),
```
Comment (PawelLipski) [inline uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala:63]: @lukaszwawrzyk @MichalBednarz possibly a bug in ide-probe

File: uiTests/src/test/resources/ide.rhino.js [hunk 4]
```diff
@@ -42,7 +42,16 @@ function Ide() {
   this.soleOpenedProject = function () {
     const openProjects = ProjectUtil.getOpenProjects();
     return openProjects.length === 1 ? new Project(openProjects[0]) : null;
-  }
+  };
+
+  this.closeOpenedProjects = function () {
```
Comment (PawelLipski) [inline uiTests/src/test/resources/ide.rhino.js:47]: Restored the old (pre-ideprobe) implementation

**Review by mkondratek — APPROVED** [review]
lgtm

**Pull request merged**